components/UWAPlantMap.vue

```diff
@@ -80,10 +119,11 @@ export default {
         this.userMarker = null
       } else {
         this.userMarker.setPosition(this.position)
-      }
+      }df8
```

Comment:
what this lol

Reply:
Lol i found that on accident when the user position stopped working. Dunno how it got in.